charts/sscs-ccd-callback-orchestrator/values.yaml

```diff
@@ -27,6 +26,7 @@ java:
     LOG_OUTPUT: single
     IDAM_OAUTH2_CLIENT_ID: "sscs"
     TRUST_ALL_CERTS: "false"
+    IDAM_S2S_AUTH: "http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
```

Comment:
why not https?

Reply:
That's what we have on our other apps. Plus that's what we have currently on ASE. Wary of making too many changes from ASE to AKS in case it breaks.